src/frontends/onnx/README.md

```diff
@@ -0,0 +1,54 @@
+# ONNX FE
+
+The main responsibility of ONNX Frontend is importing ONNX models and conversion them to the `ov::Model` representation. 
```

Comment:
Either 'importing and converting' or 'import of and conversion of'.
Rather use 'these' while referring to non-living matter.
My choice would be 'import of ONNX models and conversion of these into the ...'